src/SingleCellAnnotator.imjoy.html

```diff
@@ -34,7 +34,7 @@
 
 <script lang="javascript">
 const SERVERS = {
-  'production': 'https://wet-grasshopper-41.telebit.io',
+  'production': 'https://witty-grasshopper-61.telebit.io',
```

Comment:
dont forgot to git merge the latest master before merging. I updated covid-annotaiton...html

Reply:
I created a new PR because of permission error